src/screens/SuperAdminOrgPage/SuperAdminOrgPage.tsx

```diff
@@ -1,8 +1,26 @@
 import React from 'react';
 import styles from './SuperAdminOrgPage.module.css';
 import AdminNavbar from 'components/AdminNavbar/AdminNavbar';
+import { gql, useQuery } from '@apollo/client';
+import OrganizationCard from 'components/OrganizationCard/OrganizationCard';
 
 function SuperAdminOrgPage(): JSX.Element {
+  const ORGANIZATION_LIST = gql`
+    query {
+      organizations {
+        image
+        _id
+        creator {
+          firstName
+          lastName
+        }
+        name
+      }
+    }
+  `;
```

Comment:
Although this query is really short, please create a queries folder to house your grpahQL queries, and import them into your components.

So this query could be found in `queries/superAdmin.js`

Reply:
ok sir